app/views/exports/hamilton_lca_pbmc_bank_to_lca_bank_stock.csv.erb

```diff
@@ -22,15 +22,20 @@
   @plate.wells_in_columns.each do |source_well|
     next if source_well.empty?
 
-    # assume 1 tube per source_well (many source_wells to 1 tube)
+    # expecting 1 downstream tube for each source plate well
+    next if source_well.downstream_tubes.empty?
+
```

Comment:
So will this now create an empty file if they have not yet created the downstream tubes?